### PR TITLE
Feature/submit button

### PR DESCRIPTION
**Pull request by MKaHead**
**General:**
-  Change all proposal class properties to static get methods
- Fix global collision css coming from the controlled input demos.

**Button Component**
- Button consumes now the icon component as dependency etc.
- Implement submit functionality
- Add more stories for submit functionality in native stories and react demo
- Import webcomponents polyfill, because every component is plug and play
- Improve React Wrapper - set default values for the props, because the webcomponent constructor don't set default values of the React props coming in.
- Add defineOnce function

**Icon Component**
- Add defineOnce function
- Import webcomponents polyfill and as dependency, because every component is plug and play

**Storybook**
- Update Storybook version

### CODE REVIEW

File: src/components/10-atoms/icon/index.js [hunk 1]
```diff
@@ -28,11 +29,10 @@ class AXAIcon extends LitElement {
     'upload': UploadSvg
   };
 
-  static get properties() {
-    return {
-      icon: { type: String },
-    };
-  }
+  static properties = {
```
Comment (raphaellueckl) [inline src/components/10-atoms/icon/index.js]: get (as discussued)

File: src/demo/demo-controlled-inputs/App.css [hunk 2]
```diff
@@ -16,18 +16,18 @@
   font-size: 16px;
 }
 
-article {
-  padding: 1rem;
-  box-shadow: 1px 2px 3px #ddd;
-}
+/*article {*/
```
Comment (raphaellueckl) [inline src/demo/demo-controlled-inputs/App.css]: Pls remove outcommented code. If we need it in the future, we have version control to see how it was.

File: src/components/10-atoms/icon/index.js (hunk 1 above)
Comment (MKaHead) [reply src/components/10-atoms/icon/index.js]: fixes #872ae70

File: src/demo/demo-controlled-inputs/App.css (hunk 2 above)
Comment (MKaHead) [reply src/demo/demo-controlled-inputs/App.css]: fixes in #f027467

**Pull request merged**